app/models/expenditures_with_circ_stats_report.rb

```diff
@@ -34,4 +35,15 @@ def write_lib
   def write_fmt
     self[:formats] = format_array.delete_if { |a| a.empty? || a == 'All Formats' }.join(',')
   end
+
+  def start_date_present?
+    case date_type
+    when 'calendar'
+      cal_start.present?
+    when 'fiscal'
+      fy_start.present?
+    when 'paydate'
+      pd_start.present?
+    end
```

Comment:
Add default start date instead of returning error message?

Reply:
ditto 👆